src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,14 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        Machine[] cars = new Machine[] {new Excavator(), new Truck(), new Bulldozer()};
+
+        for (Machine car : cars) {
```

Comment:
I think 'car' is not an appropriate enough word. It's better to name it machine/machines